weaviate/values.yaml

```diff
@@ -37,6 +37,27 @@ args:
 #   - "-c"
 #   - "ulimit -n 65535 && /bin/weaviate --host 0.0.0.0 --port 8080 --scheme http --config-file /weaviate-config/conf.yaml"
 
+
+# it is possible to change the sysctl's 'vm.max_map_count' using initContainer for Weaviate,
+# the init Container runs before Weaviate Container and sets the value for the WHOLE node
+# to the one provided below.
+# it is possible to run additional initContainer before Weaviate is up and running. You can specify the
+# containers as a list, exactly how they are defined in a kubernetes manifest:
+#   https://kubernetes.io/docs/concepts/workloads/pods/init-containers/
+initContainers:
+  sysctlInitContainer:
+    enabled: true
+    sysctlVmMaxMapCount: 524288
+    image:
+      registry: docker.io
+      repo: alpine
+      tag: latest
+      pullPolicy: IfNotPresent
```

Comment:
we have this part configurable here, but command is in `stateful set`. Wouldn't it be better to either move all of this to `stateful set`? or put all here?

If all would be here then I would make it simpler like this:
```
initContainers:
    - name: configure-sysctl
        securityContext:
          runAsUser: 0
          privileged: true
        image: docker.io/alpine:latest
        imagePullPolicy: IfNotPresent
        command: ["sysctl", "-w", "vm.max_map_count=524288", "-w", "vm.overcommit_memory=1" ]
#    - name: additional-custom-init-container
#       image: some-image
#    - name: antoher-additional-custom-init-container
#       image: some-image

```

Reply:
The reason I did not do it like this is because I do not want users to change too much in the `configure-sysctl` init Container. It is supposed to do one thing, change the `vm.max_map_count`. The reason we have these settings:
```yaml
image:
      registry: docker.io
      repo: alpine
      tag: latest
      pullPolicy: IfNotPresent
```
is because some users might not want to pull data from public repositories.

Reply:
ok, then we can leave it as it is now